plugins/chakra-ui/tsconfig.json

```diff
@@ -0,0 +1,9 @@
+{
+    "compilerOptions":{
+       "paths": {
+          "@chakra/*":[
+             "src/definitions/chakra/*"
+          ]
+       }
```

Comment:
bence bu şekilde explicit yapmayalım, @definitions yapsak yeterli

Reply:
yapisal olan pathleri, template de versek daha dogru olmaz mi ? 

components, definitions, types gibi pathleri template uzerinden verebiliriz. 

@styled gibi @chakra icinden de farkli tema ve typelar cikartilabilir diye ekledim aslinda bu yolu da, kaldirabiliriz

Reply:
tabi her plugin in`"@definitions/*": ["src/definitions/*"]` eklemesinde  bir sakinca yok, tekrar edenleri eklemeyecek merge yaparken

Reply:
Bence bu kadar detayli path'lere gerek yok. Definitions vs eklesek bence yeterli